mass_mailing_switzerland/models/mass_mailing.py

```diff
@@ -86,14 +86,19 @@ def send_mail(self):
             )
             emails += super(MassMailing, mailing).send_mail()
 
+        emails_set = {}
```

Comment:
Here you are creating a dictionary, not a set. You should use `set()` if you want an empty set.